src/Horde.jl

```diff
@@ -1,38 +1,40 @@
 module Horde
 import Base.get
 
-export GVF, PredictionGVF, get_parameters, get,
+export GVF, GVFHorde, get_parameters, get,
     FeatureCumulant, ConstantDiscount, NullPolicy
 
 include("GVF/GVF.jl")
 
-
-mutable struct GVFHorde{T<:AbstractGVF}
-    gvfs::Array{T, 1}
+abstract type AbstractHorde end
+
+# Previous Implementation
```

Comment:
Old code can always be found in the git history (especially if commit messages are descriptive). I normally find adding commented code to the repository makes things messy quickly.

Reply:
Just habits.

Reply:
Thanks for the comment though! I'll remember this for the future.